bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/i18n/awattar_de.properties

```diff
@@ -167,7 +167,7 @@ error.json=Ungültiges JSON von aWATTar
 error.interrupted=Kommunikation unterbrochen
 error.execution=Ausführungsfehler
 error.timeout=Timeout beim Abrufen der Preise von aWATTar
-error.invalid.data=Keine oder ungültige Daten von der aWATTar API erhalten
+error.empty.data=Keine oder ungültige Daten von der aWATTar API erhalten
```

Comment:
Please remove changes to this file, translations should be provided by crowdin.

Reply:
Is this all there? I thought i should only change it in the english version as @J-N-K mentioned (https://github.com/openhab/openhab-addons/pull/17169#discussion_r1694271107)

(sorry i do not know CrowdIn)

Reply:
Only make changes to the default language (english). Others can be contributed by crowdin.

Reply:
Hi @tl-photography  I used it long time ago, so I can't remember exactly, but AFAIR: Log in to https://crowdin.com/ using GitHub. In my case openhab-addons is already present as project, I don't know whether this will be the case for you, too, but as soon as it is, i remember it was very straightforward to add translations there.

Reply:
Alright, will check it out and adapt the last commit accordingly. Thanks!

Reply:
But now both files are reverted. The default should be provided/adapted the german/de variant can be adapted by crowding. hope this makes it more clear.

for reference:  https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations

Reply:
Yes, I reverted it first and now playing a bit with CrowdIn.

I will update it soon and make the rest in CrowdIn.